website/project/views/node.py

```diff
@@ -741,7 +741,7 @@ def _view_project(node, auth, primary=False):
             'alternative_citations': [citation.to_json() for citation in node.alternative_citations],
             'has_draft_registrations': node.has_active_draft_registrations,
             'contributors': [contributor._id for contributor in node.contributors],
-            'is_preprint': node.is_preprint,
+            'is_preprint': node.has_published_preprint,
```

Comment:
@mfraezz As I was doing a final pass on my PR, I noticed that you had changed `is_preprint` here to be `has_published_preprint`, when is_preprint  contains `has_published_preprint`. Can you clarify if my PR should be doing the same or was that an oversight?